src/core/config.go

```diff
@@ -443,6 +443,7 @@ type Configuration struct {
 		Sandbox         bool         `help:"Deprecated, use sandbox.test instead."`
 		DisableCoverage []string     `help:"Disables coverage for tests that have any of these labels spcified."`
 		Upload          cli.URL      `help:"URL to upload test results to (in XML format)"`
+		Gzip            bool         `help:"True to upload the test results gzipped."`
```

Comment:
Do we want to group these two as they are related?

Reply:
If we do group them, it wont be backward compatible, so I don't think that's what we want.

Reply:
Might be tempted to call this `UploadGzipped` or something so it's clear that it relates to the upload (as opposed to just `test.gzip` which could mean other things)

Reply:
Ah yup good point!